portability-worker/src/main/java/org/dataportabilityproject/worker/WorkerModule.java

```diff
@@ -45,6 +44,22 @@
     this.context = context;
   }
 
+  private static TransferExtension findTransferExtension(
```

Comment:
nit: can you move this to the bottom of the file? java style guide has private methods at the bottom of classes

Reply:
Weird, the reformatter keeps moving it up there - set it manually